### PR TITLE
chore(docs-dev-env): Add more information on integration tests

### PR DESCRIPTION
**Pull request by erezrokah**
Related to https://github.com/cloudquery/docs/issues/134

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,8 @@ cq-provider-aws
 .cq/*
 # Local .terraform directories
 **/.terraform/*
+# Local .terraform.lock.hcl files
+**/terraform/**/local/.terraform.lock.hcl
```
Comment (erezrokah) [inline .gitignore:21]: I had to run `terraform init` before `terraform apply -var="prefix=AB"`, and it generated these files.
In local dev context, I think it's ok to ignore them.

File: docs/contributing/adding_a_new_resource.md [hunk 2]
```diff
@@ -53,35 +53,53 @@ A few important things to note when adding functions that call the AWS API:
...
 * Make sure to replace built-in plain text passwords with `random_password` generator
 * For every compute/db try to use the smallest size to keep the cost low
 * If autoscaling option is present, always turn it off
 
 If you want to apply the terraform locally first before pushing it to CI and applying there use:
 
-```
+```bash
 cd terraform/YOUR_SERVICE_NAME/local
+terraform init
```
Comment (erezrokah) [inline docs/contributing/adding_a_new_resource.md:99]: See https://github.com/cloudquery/cq-provider-aws/pull/723/files#r852899059.

Please let me know if I'm missing anything here.

File: .gitignore (hunk 1 above)
Comment (zagronitay) [reply .gitignore:21]: agree
btw not all of the local envs accept the prefix var
maybe we should add it

File: docs/contributing/adding_a_new_resource.md [hunk 3]
```diff
@@ -53,35 +53,53 @@ A few important things to note when adding functions that call the AWS API:
...
 # Use AB as your initial so you can have multiple team members working on the same account without conflicting resources
 terraform apply -var="prefix=AB"
 go test -run="TestIntegration/ROOT_TABLE_NAME" -tags=integration ./...
+```
+
+>For CloudQuery employees, please use the Playground AWS account to deploy and test your resources. Make sure to destroy the resources once you're done.
```
Comment (zagronitay) [inline docs/contributing/adding_a_new_resource.md:105]: I don't know if I will add this here

Comment (erezrokah) [reply docs/contributing/adding_a_new_resource.md:105]: Cool, I'm good with removing it

File: .gitignore (hunk 1 above)
Comment (erezrokah) [reply .gitignore:21]: I can a separate issue for supporting the prefix var in all local envs so we can do it in another PR.
Does that work?

File: docs/contributing/adding_a_new_resource.md [hunk 4]
```diff
@@ -53,35 +53,51 @@ A few important things to note when adding functions that call the AWS API:
 
 ### Integration Tests
 
-To Run Integration tests please run
-
+To prepare your environment for running integration tests:
 ```bash
 docker run -p 5432:5432 -e POSTGRES_PASSWORD=pass -d  postgres:13.3
```
Comment (erezrokah) [inline docs/contributing/adding_a_new_resource.md:58]: Also - a question. Do we need to specify the `postgres` version for this to work?
CI uses latest https://github.com/cloudquery/cq-provider-aws/blob/8603791b3bbfbd924ade4271cb5472f98418d7a7/.github/workflows/test_integration.yml#L44 so maybe we can do the same here?

Comment (roneli) [reply docs/contributing/adding_a_new_resource.md:58]: mock tests run vs 11,12, 13. Latest is using version 14 which is okay. We support >11. I don't think its useful to run e2e tests on all versions, mock tests is enough and usually faster.

**Review by roneli — APPROVED** [review]
LGTM, anything blocking merging this?

**Comment by erezrokah** [comment]
>LGTM, anything blocking merging this?

Nope, good to go now

**Pull request merged**